crates/virtio-queue/CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+# v0.7.0
+
+# Changed
+
+- Updated vmm-sys-util from 0.10.0 to 0.11.0
```

Comment:
Can you please update all your changes in CHANGELOGs to end with full stop?

Reply:
Done